crates/sui-pg-db/src/lib.rs

```diff
@@ -217,10 +228,17 @@ async fn pool(
 ) -> anyhow::Result<Pool<AsyncPgConnection>> {
     let statement_timeout = args.statement_timeout();
 
+    // Build TLS configuration once
+    let tls_config = build_tls_config(&args)?;
+
     let mut config = ManagerConfig::default();
+
     config.custom_setup = Box::new(move |url| {
+        let tls_config = tls_config.clone();
+        let statement_timeout = statement_timeout;
```

Comment:
My intuition for this is famously bad and I usually keep just trying things until they work, but I couldn't figure out why we need this -- what happens when you don't include it?

My thinking is that the outer closure has `move`, `tls_config` is not used outside of this closure, and `statement_timeout` has `Copy`, so it should be fine to refer to `statement_timeout` directly inside the `move async` block, and maybe for tls_config you need to do `let tls_config = tls_config;` but you shouldn't need to clone it.